src/ceph-volume/ceph_volume/devices/lvm/deactivate.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import logging
+import sys
+from textwrap import dedent
+from ceph_volume import conf
+from ceph_volume.util import encryption, system
+from ceph_volume.api.lvm import get_lvs_by_tag
+
+logger = logging.getLogger(__name__)
+
+
+def deactivate_osd(osd_id=None, osd_uuid=None):
+
+    lvs = []
+    if osd_uuid:
+        lvs = get_lvs_by_tag('ceph.osd_fsid={}'.format(osd_uuid))
+    else:
+        lvs = get_lvs_by_tag('ceph.osd_id={}'.format(osd_id))
+
+    try:
+        data_lv = next(lv for lv in lvs if lv.tags['ceph.type'] in ['data', 'block'])
+    except StopIteration:
+        logger.error('No data or block LV found for OSD {}'.format(osd_id))
+        sys.exit(1)
+    conf.cluster = data_lv.tags['ceph.cluster_name']
+    logger.debug('Found cluster name {}'.format(conf.cluster))
+
+    tmpfs_path = '/var/lib/ceph/osd/{}-{}'.format(conf.cluster, osd_id)
+    logger.info('Unmounting osd-{}\'s tmpfs path at {}'.format(
```

Comment:
nit: this log message could go into umount_tmpfs() so that it only prints if it is actually tmpfs?